srcpkgs/sview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'sview'
+pkgname=sview
+version=20.08
+revision=1
+archs="i686 x86_64"
+wrksrc="sview-20_08"
+hostmakedepends="codeblocks ffmpeg libopenal libavcodec libavdevice libavformat libavutil libswscale libwebp libconfig++ libconfig libXpm pkg-config gtk+ libXrandr"
+makedepends="ffmpeg-devel libopenal-devel libwebp-devel libconfig++-devel libconfig-devel libXpm-devel gtk+-devel libXrandr-devel"
+depends="libopenal libavcodec libavdevice libavformat libavutil libswscale libwebp libconfig++ libconfig libXpm"
+short_desc="Stereoscopic video player"
+maintainer="Pavel Pivovarov <pavel.pivovarov@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://sview.ru/en"
+distfiles="https://github.com/gkv311/sview/archive/20_08.tar.gz"
```

Comment:
Use `${version//./_}`, so no need to change on update, same for wrksrc.

Reply:
Oh, that's a nice trick :+1: thanks!